### DISCUSSION TITLE
Is there a way to only print the hash value ?

### DISCUSSION THREAD

**Discussion by akshaim** [rhash/RHash#215] [Q&A] [answered]
Hi :)
I wanted to check if there was a way to print **just** the hash value ? I tried using `--printf` but I am not able to use it as it shows `incompatible formatting options` error.  I tried using ` --simple` option but it also shows the file name, which I don't need.

Help is appreciated :)

**Comment by rhash** [comment]
Hello, I can't reproduce your case. The "incompatible formatting options" error is printed when several formatting options are specified.

**Comment by rhash** [comment, marked as answer]
Duplicate of #217, see also FR #259.
